### PR TITLE
Check for ready condition not true

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Check for ready condition not true instead of just false.

/cc @losipiuk

### CODE REVIEW

File: cluster-autoscaler/clusterstate/clusterstate.go [hunk 1]
```diff
@@ -773,7 +773,7 @@ func buildScaleDownStatusClusterwide(candidates map[string][]string, lastProbed
 func isNodeNotStarted(node *apiv1.Node) bool {
```
Comment (losipiuk) [inline cluster-autoscaler/clusterstate/clusterstate.go]: nit: If you are changing this part of code consider removing Not from name of this function.
Maybe call it "isNodeStillStarting" or "isNodeStarted" negating the semantics.

File: cluster-autoscaler/clusterstate/clusterstate_test.go [hunk 2]
```diff
@@ -722,3 +722,35 @@ func TestGetClusterSize(t *testing.T) {
 	assert.Equal(t, 3, currentSize)
 	assert.Equal(t, 30, targetSize)
 }
+
+func TestIsNodeNotStarted(t *testing.T) {
+	testCases := []struct {
+		desc         string
+		condition    apiv1.NodeConditionType
+		status       apiv1.ConditionStatus
+		recentResult bool
```
Comment (losipiuk) [inline cluster-autoscaler/clusterstate/clusterstate_test.go]: name this "expectedResult"

File: cluster-autoscaler/clusterstate/clusterstate.go (hunk 1 above)
Comment (aleksandra-malinowska) [reply cluster-autoscaler/clusterstate/clusterstate.go]: Done

File: cluster-autoscaler/clusterstate/clusterstate_test.go [hunk 3]
```diff
@@ -722,3 +722,35 @@ func TestGetClusterSize(t *testing.T) {
 	assert.Equal(t, 3, currentSize)
 	assert.Equal(t, 30, targetSize)
 }
+
+func TestIsNodeNotStarted(t *testing.T) {
```
Comment (losipiuk) [inline cluster-autoscaler/clusterstate/clusterstate_test.go]: rename the the test method to match renamed name :)

**Comment by losipiuk** [comment]
LGTM

File: cluster-autoscaler/clusterstate/clusterstate_test.go (hunk 3 above)
Comment (aleksandra-malinowska) [reply cluster-autoscaler/clusterstate/clusterstate_test.go]: Done, thanks for catching this

**Pull request merged**